akka-javasdk-tests/src/test/java/akkajavasdk/components/workflowentities/TransferWorkflow.java

```diff
@@ -45,17 +45,17 @@ public Effect<Message> startTransfer(Transfer transfer) {
     } else {
       if (currentState() == null) {
         return effects()
-          .updateState(new TransferState(transfer, "started"))
-          // FIXME: use method ref and make it typed
-          .transitionTo(withdrawStepName, new Withdraw(transfer.from(), transfer.amount())) 
-          .thenReply(new Message("transfer started"));
+            .updateState(new TransferState(transfer, "started"))
+            .call(TransferWorkflow::withdraw)
+            .withInput(new Withdraw(transfer.from(), transfer.amount()))
```

Comment:
This needs now to be done in two step, like in the component client.

We can keep the `transtionTo`, but it feels more natural IMO if we make it explicit that a call to a method with a input will be done.